### DISCUSSION TITLE
Security Vulnerability

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1320] [Q&A] [answered]
WordFence is reporting a security vulnerability with the plugin.

It states: "The 12 Step Meeting List plugin for WordPress is vulnerable to unauthorized access due to a missing capability check in versions up to, and including, 3.14.26. This makes it possible for authenticated attackers, with subscriber-level access and above, to perform an unauthorized action."

See https://www.wordfence.com/threat-intel/vulnerabilities/wordpress-plugins/12-step-meeting-list/12-step-meeting-list-31426-missing-authorization

**Comment by dvcb** [comment]
Thank you @Grateful-Viv .  We're still trying get details on this potential issue. We'll fix any issue as soon as possible.

**Reply by Grateful-Viv** [reply]
Looks like 3.14.27 was released and there should be no more warnings. Thanks all for the fast work.

**Reply by Grateful-Viv** [reply]
WordFence is at it again. 

See https://www.wordfence.com/threat-intel/vulnerabilities/wordpress-plugins/12-step-meeting-list/12-step-meeting-list-31426-missing-authorization

**Comment by kiyote33** [comment]
Release 3.14.29 appears to have reversed the warnings on Wordfence.

**Reply by Grateful-Viv** [reply]
WordFence just sent out notification about [3.14.29](https://www.wordfence.com/threat-intel/vulnerabilities/wordpress-plugins/12-step-meeting-list/12-step-meeting-list-31426-missing-authorization ).

**Reply by dvcb** [reply]
Thanks for letting us know, @Grateful-Viv .  I confirmed that another site running Wordfence is also showing that same warning.  I reached out to Wordfence to see if they will update their database.  With this issue they are copying their vulnerability data from Patchstack who originally reported the concern.  Patchstack's site is showing the issue is resolved, so maybe Wordfence's database is just behind.

**Reply by kiyote33** [reply]
FYI: A new Wordfence Scan operation on your website should remove the warning once Patchstack shows the issue resolved. It did for me anyway...

**Reply by dvcb** [reply]
As of 9:30 AM PST 2/7 Wordfence Pro is [still complaining that 3.14.29 has a problem](https://www.wordfence.com/threat-intel/vulnerabilities/wordpress-plugins/12-step-meeting-list/12-step-meeting-list-31426-missing-authorization).    Meanwhile [Patchstack's site ](https://patchstack.com/database/vulnerability/12-step-meeting-list/wordpress-12-step-meeting-list-plugin-3-14-26-broken-access-control-vulnerability)confirms the issue is resolved.  I emailed Wordfence yesterday morning asking them to look into it but haven't heard back yet.

**Reply by kiyote33** [reply, marked as answer]
@dvcb reports "This morning I heard back from Wordfence and they have updated their database.  I ran another scan with Wordfence Pro and it no longer complains about a vulnerability."
